torch/_inductor/decomposition.py

```diff
@@ -1150,3 +1150,19 @@ def rrelu_with_noise_functional(
     else:
         negative_slope = (lower + upper) / 2
         return aten.leaky_relu(self, negative_slope), torch.Tensor()
+
+
+@register_decomposition(aten.repeat_interleave.Tensor)
+def repeast_interleave_Tensor(
```

Comment:
cc @desertfire @benjaminglass1 this is failing in aoti autotuning because it requires an input with a specific tensor data to run properly - repeat.sum() == output_size iirc.

Is there a mechanism to register a fn that will be used to generate the data in autotuning ?